README.md

```diff
@@ -1 +1,36 @@
-# HELLOMICROVERSE
\ No newline at end of file
+# Set-up-a-Hello-Microverse-project
+
+# Description the project.
+In this project, I will set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well, so it is important to know them!
+
+# Built With
+
+    html 5 , css
+    visual studio code
+    
```

Comment:
- [ ] I suggest you to keep readme professional. Be attentive to spelling and identation.  In case of built with section, I can see a space before comma, and css html written with small letters. Please upcase them and make them look as a list:

```
## Built With
   -  HTML5
   -  CSS3
   -  Visual Studio Code
```